### PR TITLE
Add Venue resource

### PR DESCRIPTION
**Pull request by illia108**
Hi. Please review changes @sveredyuk @sergiy-koshoviy @skrypalyk @yuriichaban

### CODE REVIEW

File: app/models/venue.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Venue < ApplicationRecord
+  extend FriendlyId
+  friendly_id :name, use: :slugged
+
+  has_many :events
+
+  validates :name, presence: true
```
Comment (sveredyuk) [inline app/models/venue.rb]: fengshui style
```ruby
validates :name,     presence: true
validates :address,  presence: true
validates :maps_url, presence: true
```

File: config/routes.rb [hunk 2]
```diff
@@ -5,12 +5,14 @@
 
   #=== MAIN APP =================================
   resources :events, only: %i[index show]
+  resources :venues, only: %i[index show]
```
Comment (sveredyuk) [inline config/routes.rb]: `only: :show`, we don't need index for venues at main app

File: db/migrate/20161203210855_create_venues.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreateVenues < ActiveRecord::Migration[5.0]
+  def change
+    create_table :venues do |t|
+      t.string :name
+      t.string :address
+      t.string :maps_url
```
Comment (sveredyuk) [inline db/migrate/20161203210855_create_venues.rb]: sorry.. i made typo in issue description `maps_url`. Could you please change to `map_url` ?

Comment (illia108) [inline db/migrate/20161203210855_create_venues.rb]: should I make another migration?

Comment (sveredyuk) [inline db/migrate/20161203210855_create_venues.rb]: No, rollback one step, change migration, migrate again. And fix all related code

**Comment by illia108** [comment]
I've made changes mentioned above @sveredyuk

**Comment by illia108** [comment]
Routes conflict is fixed @sveredyuk

**Pull request merged**